src/server/instrument.ts

```diff
@@ -0,0 +1,41 @@
+// This file should be imported before any other modules
+import * as Sentry from '@sentry/node';
+import { settings } from './config.js';
+
+// Initialize Sentry before any other imports
+Sentry.init({
+  dsn: settings.SENTRY_DSN,
+  integrations: [
+    Sentry.httpIntegration({
+      ignoreIncomingRequests: (url) => {
+        return (
+          url.includes('/health') ||
+          url.includes('/_next') ||
+          url.includes('/assets')
+        );
+      },
+    }),
+    Sentry.expressIntegration(),
+  ],
+  tracesSampleRate: settings.NODE_ENV === 'production' ? 0.05 : 0.1,
```

Comment:
You sure to capture only around 5% on production?

Reply:
yes, true this one is for performance monitoring which we may not needed for this app.